lib/getLatestRelease.js

```diff
@@ -18,9 +18,18 @@ const getOS = str => {
     includes(str, 'mac')
   ) {
     return 'macOS';
-  } else if (includes(str, 'win32') || includes(str, '.exe')) {
+  } else if (
+    includes(str, 'win32') ||
+    includes(str, 'windows') ||
```

Comment:
I guess it's a long time since downloads included `Xwindows` in their filenames... :laughing: